bin/host/src/interop/handler.rs

```diff
@@ -104,7 +105,7 @@ impl HintHandler for InteropHintHandler {
                 // Fetch the blob sidecar from the blob provider.
                 let mut sidecars = providers
                     .blobs
-                    .fetch_filtered_sidecars(&partial_block_ref, &[indexed_hash])
+                    .fetch_filtered_sidecars(&partial_block_ref, [indexed_hash].as_slice())
```

Comment:
Isn't this the same?

Reply:
Removed this + the other occurence, for some reason rust-analyzer yells at me about it but it compiles just fine.